draftivist_ui/src/models.ts

```diff
@@ -83,23 +86,30 @@ export class Campaign {
         }
     }
 
-    isSelected(id: number) : boolean {
+    isSelected(id: number): boolean {
         return this.selectedIssues.indexOf(id) > -1
     }
 
+    saveToDraft(draftSection: keyof Draft, content: string): void {
+        this.draft[draftSection] = content;
+    }
+
     static parse(campaignData: any) {
         const campaign = new Campaign(campaignData.id, campaignData.name, campaignData.description)
         campaign.issues = campaignData.issues.map((issueData: any) => {
             const issue = new Issue(issueData.id, issueData.text)
             issue.statements = issueData.statements.map((statementData: any) =>
-                new Statement(statementData.id, statementData.text))
-            return issue
-        })
+            ({
+                id: statementData.id,
+                description: statementData.text
+            })
+        )})
```

Comment:
Looks like this needs a `return issue` here, right now `campaign.issues` is an array of undefineds

Reply:
thanks!